FluentTerminal.App/Strings/en/Resources.resw

```diff
@@ -785,18 +786,20 @@ Fuzzy</comment>
     <comment>Command
 Fuzzy</comment>
   </data>
-  <data name="NewSshTab.Text" xml:space="preserve">
-    <value>New SSH tab</value>
+  <data name="NewRemoteTab.Text" xml:space="preserve">
```

Comment:
this has been renamed to NewSshTab with #410

Reply:
Again, I don't understand what you're telling me. I don't see `NewSshTab` control anywhere in your master. Only `NewRemoteTab` control in Main.xaml.

Reply:
Fixed.